### ISSUE TITLE
getNumericValue is null when locale is fa-IR

### ISSUE THREAD

**Issue by moeindev** [CottaCush/CurrencyEditText#5]

**Comment by Aiuspaktyn** [comment]
Same problem when the locale is italian, the problem is when there is the "," instead of "." for decimal...

a fast fix:

```
fun getNumericValue(): Double? {
        val tempParse = parseMoneyValue(text.toString(), textWatcher.decimalFormatSymbols.groupingSeparator.toString(), currencySymbolPrefix)
        return tempParse.toDoubleOrNull() ?: tempParse.replace(",", ".").toDoubleOrNull()
    }
```

**Comment by rasheedsulayman** [comment]
Nice suggestion @Aiuspaktyn. 
The issue can also be fixed by formatting output values from the `EditText` with a `NumberFormatter` created with the user supplied locale. The fix is in #6
